types/recharts/index.d.ts

```diff
@@ -757,6 +765,13 @@ export interface YPadding {
 	top: number;
 	bottom: number;
 }
+export interface YLabel {
```

Comment:
Is there any difference between `XLabel` and `YLabel`? I can't see any... How about one interface `Label` if there are indeed no differences?

Reply:
I will make those changes.